cds_ils/migrator/default_records.py

```diff
@@ -15,12 +15,15 @@
 from invenio_app_ils.ill.api import LIBRARY_PID_TYPE
 from invenio_app_ils.ill.proxies import current_ils_ill
 from invenio_app_ils.proxies import current_app_ils
+from invenio_app_ils.series.api import SERIES_PID_TYPE
 from invenio_db import db
 from invenio_pidstore.models import PersistentIdentifier, PIDStatus
 
 MIGRATION_DOCUMENT_PID = "DOCPID-44444"
 MIGRATION_LIBRARY_PID = "LIBPID-44444"
 MIGRATION_VENDOR_PID = "VENPID-44444"
+MIGRATION_YELLOW_PID = "YELLO-44444"
+MIGRATION_DESIGN_PID = "DESIG-55555"
```

Comment:
Very very minor, just for fun, what about having them like:
```suggestion
MIGRATION_DOCUMENT_PID = "L1t3r-4tUr3"
...
MIGRATION_YELLOW_PID = "Y3lL0-R3p9R"
MIGRATION_DESIGN_PID = "d3S1g-r3P0r"
```
:)